packages/core/src/components/ConditionalWrapper.js

```diff
@@ -7,13 +7,14 @@ import PropTypes from 'prop-types'
 //     wrapper: (children: JSX.Element) => JSX.Element
 //   }
 
-const ConditionalWrapper = ({ children, condition, wrapper }) =>
-    condition ? cloneElement(wrapper(children)) : children
+export const ConditionalWrapper = ({ children, condition, wrapper }) => {
+    if (!condition) return children
+
+    return cloneElement(wrapper, {}, children)
```

Comment:
The only downside to this is now you can't control the location of an element without a custom component, but also I don't know if we will be using this component that much.

Example:

```jsx
<ConditionalWrapper condition wrapper={<div><label>Example</label></div>}>
  <input type="text" />
</ConditionalWrapper>
```

In the above, the children prop will go to the div, when we want it to be a child of the label. Could make a custom component that receives the children prop to solve it. Like I said, not sure if we will be using this component extensively to worry about it.

Reply:
I think it's fine to use a custom component if we want this kind of nesting, I think it's mostly useful for the `Container`, I also changed this because I struggled to get proper typings for it.